roles/openshift_logging_fluentd/defaults/main.yml

```diff
@@ -1,12 +1,12 @@
 ---
 ### General logging settings
-openshift_logging_fluentd_image_pull_secret: "{{ openshift_hosted_logging_image_pull_secret | default('') }}"
+openshift_logging_fluentd_image_pull_secret: "{{ openshift_logging_image_pull_secret }}"
 openshift_logging_fluentd_master_url: "https://kubernetes.default.svc.{{ openshift.common.dns_domain }}"
 openshift_logging_fluentd_namespace: openshift-logging
 
 ### Common settings
 # map_from_pairs is a custom filter plugin in role lib_utils
-openshift_logging_fluentd_nodeselector: "{{ openshift_hosted_logging_fluentd_nodeselector_label | default('logging-infra-fluentd=true') | map_from_pairs }}"
+openshift_logging_fluentd_nodeselector: "{{ 'logging-infra-fluentd=true'| map_from_pairs }}"
```

Comment:
this could just be `openshift_logging_fluentd_nodeselector: { 'logging-infra-fluentd': 'true' }`, no need to use a plugin